addons/web/static/tests/views/graph_view_tests.js

```diff
@@ -3427,8 +3427,7 @@ QUnit.module("Views", (hooks) => {
                                 target: "current",
                                 type: "ir.actions.act_window",
                                 views: [
-                                    [false, "list"],
-                                    [false, "form"],
+                                    [false, "list"]
```

Comment:
Instead of deleting these lines in all test cases, could you add a form view in the model?
apply above for all test case here